### PR TITLE
Probability: Class notes

### LINKED ISSUE #9: Probability: Class notes
Gostaria de tentar trabalhar no registro das formulações em probabilidade, podendo atualizar o conteúdo a cada duas aulas.

Grato desde já pela atenção,

Gabriel

### PR DESCRIPTION
**Pull request by GHSAguiar**
The development in question aims to present the main fundamentals of probability. The student Gabriel Aguiar did not seek to fit the text into the predefined body of the "Probability" chapter.

Close #9.

### CODE REVIEW

**Comment by otaviocv** [comment]
You will need a colleague to review the code. Please, provide a github username so I can add as reviewer.

**Comment by GHSAguiar** [comment]
@otaviocv

I refer @cahtoledo

File: chaps/probability-class-notes.tex [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+\section{Formulations}
+
+\begin{itemize}
+    
+\item We define like Sample space $\Omega$ a set of elementary events.
+        
+\item We define like Sigma-algebra $\Sigma$ a set of $\Omega$ subsets. $\Sigma$ shows the following properties: set of random events; $\Omega$ belongs to $\Sigma$; if A and B belong to $\Sigma$, their union and their intersection also belong.
+
+\item We define like probability a function P: $\Sigma$ $\rightarrow$ [0, 1]. P presents the following properties: P($\Omega$) = 1; if A and B are $\Sigma$ disjoint subsets, P(A $\cup$ B) = P(A) + P(B).
```
Comment (cahmtoledo) [inline chaps/probability-class-notes.tex]:
```suggestion
\item We define as probability a function P: $\Sigma$ $\rightarrow$ [0, 1]. P presents the following properties: 
\begin{enumerate}
\item P($\Omega$) = 1; 
\item if A and B are $\Sigma$ disjoint subsets, P(A $\cup$ B) = P(A) + P(B).
\end{enumerate}
```

File: chaps/probability-class-notes.tex [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+\begin{document}
+
+\maketitle
+
+\section{Formulations}
+
+\begin{itemize}
+    
+\item We define like Sample space $\Omega$ a set of elementary events.
```
Comment (cahmtoledo) [inline chaps/probability-class-notes.tex]:
```suggestion
\item We define as Sample space $\Omega$ a set of elementary events.
```

File: chaps/probability-class-notes.tex [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        
+\item We define like Sigma-algebra $\Sigma$ a set of $\Omega$ subsets. $\Sigma$ shows the following properties: set of random events; $\Omega$ belongs to $\Sigma$; if A and B belong to $\Sigma$, their union and their intersection also belong.
```
Comment (cahmtoledo) [inline chaps/probability-class-notes.tex]:
```suggestion
\item We define as Sigma-algebra $\Sigma$ a set of $\Omega$ subsets. $\Sigma$ shows the following properties:
\begin{enumerate}
    \item set of random events;
    \item $\Omega$ belongs to $\Sigma$;
    \item if A and B belong to $\Sigma$, their union and their intersection also belong.
\end{enumerate}
```

File: chaps/probability-class-notes.tex [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+\item We define like probability a function P: $\Sigma$ $\rightarrow$ [0, 1]. P presents the following properties: P($\Omega$) = 1; if A and B are $\Sigma$ disjoint subsets, P(A $\cup$ B) = P(A) + P(B).
+
+\item P(B/A) = $\frac{1}{P(A)}$ P(A $\cap$ B).
```
Comment (cahmtoledo) [inline chaps/probability-class-notes.tex]:
```suggestion
\item P(B/A) $\doteq \frac{1}{P(A)}$ P(A $\cap$ B).
```

**Comment by GHSAguiar** [comment]
@cahmtoledo 

My choice not to integrate directly into the chapter was because I have my reservations about how it is structured. So I chose to keep my text separate for Renato to rate and see how he prefers to use his content.

**Comment by otaviocv** [comment]
Do not be afraid to add your contributions to the book it self, we will improve a little at time. It is better to have something and start from it. Let's wait for @renatovicente considerations.

**Comment by GHSAguiar** [comment]
I appreciate your consideration, @otaviocv . Thanks again for the support.

File: chaps/probability-class-notes.tex (hunk 4 above)
Comment (GHSAguiar) [reply chaps/probability-class-notes.tex]: Can you explain more about this suggestion?

**Pull request closed without merging**